### PR TITLE
Fix 11141 - replaced R_ASM_BUFSIZE with RStrBuf*

### LINKED ISSUE #11141: Kill R_ASM_BUFSIZE
screen shot 2018-08-19 at 05 35 23

### PR DESCRIPTION
**Pull request by wargio**

### CODE REVIEW

**Comment by wargio** [comment]
Fix for https://github.com/radare/radare2/issues/11141

**Comment by Maijin** [comment]
@wargio there are also 2 here if you can take care of those - https://github.com/radare/radare2-extras/search?utf8=%E2%9C%93&q=R_ASM_BUFSIZE&type=

File: libr/asm/arch/wasm/wasm.c [hunk 1]
```diff
@@ -184,11 +184,12 @@ static WasmOpDef opcodes[256] = {
...
 };
 
+// assembles the given line of wasm assembly.
 R_IPI int wasm_asm(const char *str, unsigned char *buf, int buf_len) {
 	// TODO: add immediates assembly
 	int i = 0, len = -1;
-	char tmp[R_ASM_BUFSIZE];
-	while (str[i] != ' ' && i < buf_len) {
+	char tmp[256];
+	while (str[i] && str[i] != ' ' && i < 256) {
```
Comment (radare) [inline libr/asm/arch/wasm/wasm.c]: use sizeof(tmp) instead of hardcoded 256

**Comment by radare** [comment]
solve the cornflake and rebase

File: libr/asm/arch/wasm/wasm.c (hunk 1 above)
Comment (wargio) [reply libr/asm/arch/wasm/wasm.c]: oh, right

**Pull request merged**